hsmd/hsmd_wire.csv

```diff
@@ -21,6 +21,7 @@ msgtype,hsmd_init_reply,111
 msgdata,hsmd_init_reply,node_id,node_id,
 msgdata,hsmd_init_reply,bip32,ext_key,
 msgdata,hsmd_init_reply,bolt12,pubkey32,
+msgdata,hsmd_init_reply,onion_reply_secret,secret,
```

Comment:
This is a constant that we could just as well store in the DB.

Reply:
We *could*, but it is a secret which can be (ab)used to reveal our identity.  We need to derive it deterministically anyway, so it makes sense to do it here.